mlflow/models/model.py

```diff
@@ -28,6 +33,17 @@
 )
 _MLFLOW_VERSION_KEY = "mlflow_version"
 
+PyFuncInput = Union[
+    pandas.DataFrame,
+    np.ndarray,
+    scipy.sparse.csc_matrix,  # Why do we use a string format "scipy.sparse.csc_matrix" here before?
```

Comment:
Raising a question here: this was previous set as the string format of "scipy.sparse.csc_matrix" and "scipy.sparse.csr_matrix", is there any specific reason for them?